### PR TITLE
docs: import file with command

### PR DESCRIPTION
**Pull request by robertsLando**

### CODE REVIEW

File: docs/development/config-files.md [hunk 1]
```diff
@@ -197,3 +197,18 @@ In order to get your configuration file included in this library, two things mus
 1. Update the index file using `yarn run config import -i`.
 1. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
 1. Create a PR.
+
+If your device is already present in [OZW devices DB](https://github.com/OpenZWave/open-zwave/tree/master/config), you can import it by using its device id (you can find it in zwavejs2mqtt control panel) with this command:
```
Comment (AlCalzone) [inline docs/development/config-files.md]: We should also mention that the config files contains the device ID aswell, only formatted slighly diffent:
`0x0086-0x0075-0x0004` is `0086:0075:0004` over there.

Comment (AlCalzone) [inline docs/development/config-files.md]:
```suggestion
If your device is already present in the [OpenZWave devices DB](https://github.com/OpenZWave/open-zwave/tree/master/config), you can import it by using its device id (which you can find in the zwavejs2mqtt control panel) with this command: 
```

File: docs/development/config-files.md [hunk 2]
```diff
@@ -197,3 +197,18 @@ In order to get your configuration file included in this library, two things mus
 1. Update the index file using `yarn run config import -i`.
 1. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
 1. Create a PR.
+
+If your device is already present in [OZW devices DB](https://github.com/OpenZWave/open-zwave/tree/master/config), you can import it by using its device id (you can find it in zwavejs2mqtt control panel) with this command: 
+
+```bash
+npm run config -- import -s ozw -Dmid --ids 0x0086-0x0075-0x0004
```
Comment (AlCalzone) [inline docs/development/config-files.md]: the `i` parameter was removed, we auto-update the index.json now.
Also there's no need to always update `manufacturers.json`, so we should probably document that separately and remove the `m` parameter here too.

Comment (AlCalzone) [reply docs/development/config-files.md]: We should also mention the `yarn` command, which is very similar:
```
npm run config -- import -s ozw -Dd --ids 0x0086-0x0075-0x0004
yarn run config import -s ozw -Dd --ids 0x0086-0x0075-0x0004
```

File: docs/development/config-files.md [hunk 3]
```diff
@@ -197,3 +197,18 @@ In order to get your configuration file included in this library, two things mus
...
+```
+
+You can specify multiple device ids too:
+
+```bash
+npm run config -- import -s ozw -Dmid --ids 0x0258-0x1027-0x0200 0x041a-0x0008-0x0200
```
Comment (AlCalzone) [inline docs/development/config-files.md]: Same applies to this

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: docs/development/config-files.md [hunk 4]
```diff
@@ -197,3 +197,18 @@ In order to get your configuration file included in this library, two things mus
 1. Update the index file using `yarn run config import -i`.
 1. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
 1. Create a PR.
+
+If your device is already present in the [OpenZWave devices DB](https://github.com/OpenZWave/open-zwave/tree/master/config), you can import it by using its device id (which you can find in the zwavejs2mqtt control panel) with this command: 
+
+```bash
+npm run config -- import -s ozw -Dmid --ids 0x0086-0x0075-0x0004
```
Comment (robertsLando) [inline docs/development/config-files.md]:
```suggestion
npm run config -- import -s ozw -Dmd --ids 0x0086-0x0075-0x0004
# with yarn:
yarn run config import -s ozw -Dd --ids 0x0086-0x0075-0x0004
```

File: docs/development/config-files.md [hunk 5]
```diff
@@ -197,3 +197,18 @@ In order to get your configuration file included in this library, two things mus
...
+```
+
+You can specify multiple device ids too:
+
+```bash
+npm run config -- import -s ozw -Dmid --ids 0x0258-0x1027-0x0200 0x041a-0x0008-0x0200
```
Comment (robertsLando) [inline docs/development/config-files.md]:
```suggestion
npm run config -- import -s ozw -Dmd --ids 0x0258-0x1027-0x0200 0x041a-0x0008-0x0200
# with yarn:
yarn run config import -s ozw -Dd --ids 0x0258-0x1027-0x0200 0x041a-0x0008-0x0200
```

File: docs/development/config-files.md [hunk 6]
```diff
@@ -197,3 +197,22 @@ In order to get your configuration file included in this library, two things mus
 1. Update the index file using `yarn run config import -i`.
 1. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
 1. Create a PR.
+
+If your device is already present in the [OpenZWave devices DB](https://github.com/OpenZWave/open-zwave/tree/master/config), you can import it by using its device id (which you can find in the zwavejs2mqtt control panel) with this command:
```
Comment (robertsLando) [inline docs/development/config-files.md]:
```suggestion
If your device is already present in the [OpenZWave devices DB](https://github.com/OpenZWave/open-zwave/tree/master/config), you can import it by using its device id (which you can find in the zwavejs2mqtt control panel). Device config files also contain the device id but the format is different: `0x0086-0x0075-0x0004` is `0086:0075:0004`.

The command to use is:

```

File: docs/development/config-files.md [hunk 7]
```diff
@@ -195,19 +195,26 @@ Some devices spam the network with hundreds of invalid `ConfigurationCC::InfoRep
 In order to get your configuration file included in this library, two things must be done:
 
 1. Update the index file using `yarn run config import -i`.
-1. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
-1. Create a PR.
+2. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
+3. Create a PR.
```
Comment (AlCalzone) [inline docs/development/config-files.md]: FYI, if you leave it at 1./1./1., you don't have to update the numbers if something gets changed ;)

**Comment by AlCalzone** [comment]
Thanks - I figured we should mention both sources.

File: docs/development/config-files.md (hunk 7 above)
Comment (robertsLando) [reply docs/development/config-files.md]: Didn't know :open_mouth:

**Pull request merged**